maestro-cli/src/main/java/maestro/cli/util/AndroidEnvUtils.kt

```diff
@@ -92,12 +92,18 @@ object AndroidEnvUtils {
 
     private fun findCompatibleCommandLineTool(tool: String): File? {
         val path = File(androidHome, "cmdline-tools")
+
+        var thisTool = tool
+        if (EnvUtils.isWindows()){
+            thisTool = "$tool.bat"
```

Comment:
Is this always the case on Windows?

Reply:
No, all files in cmdline-tools on Windows are batch files.

Reply:
> not to "actually working"

There's a missing accompanying change to requireEmulatorBinary() that should append `.exe` to the emulator path. That might get us closer to "actually working"...